platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/reconnect/ReconnectLearnerFactory.java

```diff
@@ -31,28 +30,28 @@
  */
 public class ReconnectLearnerFactory {
     private final AddressBook addressBook;
-    private final ReconnectSettings settings;
+    private final int reconnectSocketTimeout;
     private final ReconnectMetrics statistics;
     private final ThreadManager threadManager;
     private final PlatformContext platformContext;
 
     /**
      * @param platformContext the platform context
-     * @param threadManager   responsible for managing thread lifecycles
-     * @param addressBook     the current address book
-     * @param settings        reconnect settings
-     * @param statistics      reconnect metrics
+     * @param threadManager          responsible for managing thread lifecycles
+     * @param addressBook            the current address book
+     * @param reconnectSocketTimeout reconnectSocketTimeout
```

Comment:
the unit should be indicated in this javadoc

Reply:
thanks for the hint